src/components/Calendar.vue

```diff
@@ -143,7 +127,25 @@ function extractHourAndMinutes(timeString: string) {
       </div>
```

Comment:
<div v-if="visibleEvents.length == 0">
      <span>Er is nu even niks</span>
    </div>